db/order_products.js

```diff
@@ -71,6 +71,33 @@ const removeProductFromOrder = async (orderProductId) => {
 
 }
 
+//Remove product from order by orderId
+const removeOrderProductByOrderId = async (orderId)=>{
```

Comment:
Makes sense for technical scenarios. I originally didn't add this one because I figured that orders shouldn't be altered once their completed.